tf-modules/archive/sf_event_sqs_to_db_records.tf

```diff
@@ -1,7 +1,7 @@
 locals {
   # Pulled out into a local to prevent cyclic dependencies
-  # between the IAM role, queue and lambda function.
-  sf_event_sqs_lambda_timeout = (var.rds_connection_timing_configuration.acquireTimeoutMillis / 1000) + 60
+  # between the IAM role, queue, and lambda function.
+  sf_event_sqs_lambda_timeout = var.sf_event_sqs_lambda_timeout != null ? var.sf_event_sqs_lambda_timeout : (var.rds_connection_timing_configuration.acquireTimeoutMillis / 1000) + 60
```

Comment:
I paused on the explicit comparison to null but that's the default so seems fine. I don't see how this would be undefined.